stubs/Pillow/PIL/ExifTags.pyi

```diff
@@ -1,4 +1,2 @@
-from typing import Any
-
-TAGS: Any
-GPSTAGS: Any
+TAGS: dict[int, str]
+GPSTAGS: dict[int, str]
```

Comment:
Maybe `Mapping` would be better, so that you can't change these by accident? See also `keyword.kwlist` in stdlib.

Reply:
Sure. I don't have a strong feeling either way.